lib/logstash/outputs/stomp.rb

```diff
@@ -57,11 +66,35 @@ def register
     
     connect
   end # def register
-  
-  def receive(event)
-      
 
-      @logger.debug(["stomp sending event", { :host => @host, :event => event }])
-      @client.send(event.sprintf(@destination), event.to_json)
+  public
+  def do_close
+    @logger.warn("Disconnecting from stomp broker")
+    @client.disconnect if @client.connected?
+  end # def do_close
```

Comment:
`do_close` is our internal method which calls `close` on the plugin, so to make sure we dont introduce breaking changes we should rename that method to `close`